spec/android_device_spec.rb

```diff
@@ -29,6 +31,20 @@
       expect(devices[0].serial).to eq('SH34RW905290')
       expect(devices[0].status).to eq(:ok)
     end
+
+    it 'handles an untrusted device correctly' do
+      out = <<eof
+List of devices attached
+G090G8105387008L	unauthorized
+eof
+      allow(Open3).to receive(:capture3) { [out, '', $STATUS_ZERO] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.